src/Persistence.LinnApps/ServiceDbContext.cs

```diff
@@ -1443,6 +1446,18 @@ private void BuildEdiOrders(ModelBuilder builder)
             entity.Property(d => d.SequenceNumber).HasColumnName("SEQUENCE_NUMBER");
         }
 
+        private void BuildEdiSuppliers(ModelBuilder builder)
+        {
+            var entity = builder.Entity<EdiSupplier>().ToTable("EDI_PENDING_VIEW");
+            entity.HasKey(e => e.SupplierId);
+            entity.Property(d => d.SupplierId).HasColumnName("SUPPLIER_ID");
+            entity.Property(d => d.SupplierName).HasColumnName("SUPPLIER_NAME").HasColumnType("VARCHAR2");
+            entity.Property(d => d.VendorManager).HasColumnName("VENDOR_MANAGER").HasColumnType("VARCHAR2");
+            entity.Property(d => d.VendorManangerName).HasColumnName("USER_NAME").HasColumnType("VARCHAR2");
+            entity.Property(d => d.EdiEmailAddress).HasColumnName("EDI_EMAIL_ADDRESS").HasColumnType("VARCHAR2");
+            entity.Property(d => d.NumOrders).HasColumnName("NUM_ORDERS");
+		}
```

Comment:
weird indentation? Or is that just github

Reply:
weird looks okay in visual studio